### PR TITLE
DDP-5556: Duplicate initial biopsy

### PR DESCRIPTION
**Pull request by SimoneMaiwald**
adding a deleted flag to the duplicate initial biopsy 
deleting 688 institutions and their associated medical records manually seems scary!

added the flag to all ddp_medical_record queries to ignore them

liquidbase update script to set the flag (only for institution_id < 20000 because of actual migrated pts with new added initial_biopsies)

### CODE REVIEW

**Comment by SimoneMaiwald** [comment]
@pegahtah please double check as well if I missed any queries for ddp_medical_record
Didn't change queries in unit tests

File: src/main/resources/liquibase/042-updateDB.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            <column name="deleted" type="tinyint(1)"></column>
+        </addColumn>
+    </changeSet>
+    <changeSet author="simone" id="setDeletedFlag">
+        <sql>UPDATE ddp_medical_record as m, (
+                SELECT m.medical_record_id from ddp_medical_record m
+                left join ddp_institution i on (i.institution_id = m.institution_id)
+                left join ddp_participant p on (p.participant_id = i.participant_id)
+                left join ddp_instance realm on (p.ddp_instance_id = realm.ddp_instance_id)
+                where realm.instance_name = 'Prostate' and (p.ddp_participant_id like '1771332079%' OR p.ddp_participant_id like '-763171247%') and type = 'INITIAL_BIOPSY' and i.ddp_institution_id != '1' and i.institution_id &#60; 20000) as m2
```
Comment (SimoneMaiwald) [inline src/main/resources/liquibase/042-updateDB.xml:19]: & # 6 0 ;     equals  '<'

Comment (pegahtah) [reply src/main/resources/liquibase/042-updateDB.xml:19]: I don't understand the `i.ddp_institution_id != '1'` but looks good to me

**Review by pegahtah — APPROVED** [review]

File: src/main/resources/liquibase/042-updateDB.xml (hunk 1 above)
Comment (SimoneMaiwald) [reply src/main/resources/liquibase/042-updateDB.xml:19]: all initial_biopsy institutions in gen2 got id =1
means if pt id is gen2 and institution id is not 1 it is a newly added one
Does that make sense?
@pegahtah

Comment (pegahtah) [reply src/main/resources/liquibase/042-updateDB.xml:19]: 👍

**Review by pegahtah — APPROVED** [review]

**Pull request merged**